src/commands/createNewProject/dotnetSteps/DotnetRuntimeStep.ts

```diff
@@ -16,7 +16,7 @@ export class DotnetRuntimeStep extends AzureWizardPromptStep<IProjectWizardConte
     public static async createStep(context: IProjectWizardContext): Promise<DotnetRuntimeStep> {
         if (context.targetFramework) {
             context.targetFramework = typeof context.targetFramework === 'string' ? [context.targetFramework] : context.targetFramework;
-            const runtimes = await getRuntimes(context);
+            const runtimes = (await getRuntimes(context)).reverse(); // reverse so that the latest version is first
```

Comment:
Did you verify that getRuntimes always returns them in the same order?

Reply:
Mine always does. I will look into it some more to see how we are exactly getting it.

Reply:
Yes please 👍

Reply:
I think if we are going the prompt route  as mentioned below it won't matter either way.